services/wallet/api.go

```diff
@@ -225,6 +236,7 @@ func (api *API) DeleteSavedAddress(ctx context.Context, address common.Address,
 	return err
 }
 
+// @deprecated
 func (api *API) GetPendingTransactions(ctx context.Context) ([]*transactions.PendingTransaction, error) {
```

Comment:
@Cuteivist isn't this used to fetch details for pending transactions?

Reply:
I thought, we unified API for that?